src/js/modules/infragistics.ui.editors.js

```diff
@@ -6166,6 +6398,21 @@
 			centuryThreshold: 29,
 			/* type="number" Gets/Sets difference between year in Gregorian calendar and displayed year. */
```

Comment:
This gives absolutely no description of what the option does. Is it the difference between the Gregorian calendar and the calendar in use by the application?